### ISSUE TITLE
Crash when trying sample

### ISSUE THREAD

**Issue by AndroidDeveloperLB** [andrefrsousa/SuperBottomSheet#13]
I get a crash right when trying the sample:

```
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.andrefrsousa.superbottomsheet.demo/com.andrefrsousa.superbottomsheet.demo.MainActivity}: android.view.InflateException: Binary XML file line #2 in com.andrefrsousa.superbottomsheet.demo:layout/activity_main: Binary XML file line #2 in com.andrefrsousa.superbottomsheet.demo:layout/activity_main: Error inflating class androidx.constraintlayout.ConstraintLayout
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
     Caused by: android.view.InflateException: Binary XML file line #2 in com.andrefrsousa.superbottomsheet.demo:layout/activity_main: Binary XML file line #2 in com.andrefrsousa.superbottomsheet.demo:layout/activity_main: Error inflating class androidx.constraintlayout.ConstraintLayout
     Caused by: android.view.InflateException: Binary XML file line #2 in com.andrefrsousa.superbottomsheet.demo:layout/activity_main: Error inflating class androidx.constraintlayout.ConstraintLayout
     Caused by: java.lang.ClassNotFoundException: androidx.constraintlayout.ConstraintLayout
        at java.lang.Class.classForName(Native Method)
        at java.lang.Class.forName(Class.java:454)
        at android.view.LayoutInflater.createView(LayoutInflater.java:815)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1006)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:961)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:659)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:534)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:481)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:555)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:161)
        at com.andrefrsousa.superbottomsheet.demo.MainActivity.onCreate(MainActivity.kt:39)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
2019-10-17 13:06:10.306 30684-30684/com.andrefrsousa.superbottomsheet.demo E/AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "androidx.constraintlayout.ConstraintLayout" on path: DexPathList[[zip file "/data/app/com.andrefrsousa.superbottomsheet.demo-eVeuOh4vgM5BaZM3Layqvw==/base.apk"],nativeLibraryDirectories=[/data/app/com.andrefrsousa.superbottomsheet.demo-eVeuOh4vgM5BaZM3Layqvw==/lib/arm64, /system/lib64, /system/product/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:196)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        	... 26 more
```

[2019-10-17_13-06-09.zip](https://github.com/andrefrsousa/SuperBottomSheet/files/3738616/2019-10-17_13-06-09.zip)

**Comment by AndroidDeveloperLB** [comment]
I think you should just use "androidx.constraintlayout.widget.ConstraintLayout" instead of "androidx.constraintlayout.ConstraintLayout" in the layout files.
You also need to replace "android.support.design.widget.CoordinatorLayout" with "androidx.coordinatorlayout.widget.CoordinatorLayout"

**Comment by andrefrsousa** [comment]
@AndroidDeveloperLB thanks for the crash report. It should be fixed in `1.5.0`.

**Comment by AndroidDeveloperLB** [comment]
Thank you. Speaking of samples, can you please put more ?
Maybe one with RecyclerView, showing a grid of items like the share-bottom sheet?
Recently it got even more special, as items expand and collapse... A bit hard to explain...
